tv-shows-app/src/app/components/feature/review/ReviewItem/ReviewItem.tsx

```diff
@@ -0,0 +1,29 @@
+import { IReview, IReviewItem } from "@/typings/review";
+import { Button, Card, CardBody, Flex, Show, Text } from "@chakra-ui/react";
+import { Fragment } from "react";
+import ReviewStarsInput from "../ReviewStarsInput/ReviewStarsInput";
+import { IStar } from "@/app/components/shared/Icons/StarIcon/StarIcon";
+
+export interface IReviewItemProps {
+   review: IReview,
+   onDelete: (review: IReview) => void;
+}
+
+export default function ReviewItem({review, onDelete} : IReviewItemProps) {
+   const value : IStar[] = [];
+   for (let i = 0; i < 5; i++) value.push({selected: i < review.rating});
+
+   return <Card padding={1} backgroundColor={'lightblue'} color={'white'}>
+      <Flex direction={'column'} gap = {1}>
+         <Text> {review.text} </Text>
+         <Text> {`${review.rating} / 5`} </Text>
+         <ReviewStarsInput value={value} onChange={() => {return;}}></ReviewStarsInput>
+         <Show above='768px'>     

```

Comment:
Super što si skužila ove media-query propove u chakri!
Rijetko kad ćemo pisati `above='768px',` obično ćemo i za ovo koristiti predefinirane tokene koje već Chakra nudi tipa:
` <Show above='md'>     `